pg2kinesis/formatter.py

```diff
@@ -156,7 +158,7 @@ class CSVPayloadFormatter(Formatter):
     VERSION = 0
     def produce_formatted_message(self, change):
         fmt_msg = '{},{},{}'.format(CSVFormatter.VERSION, CSVFormatter.TYPE,
-                                    json.dumps(change.__dict__))
+                                    json.dumps(change._asdict()))
```

Comment:
Why this?

Reply:
I had explained this in a previous comment, but it looks like it was hidden. `__dict__` is not supported on NamedTuple types in Python3. I'm not super happy about calling a `_leadingunderscore` named method, but it is the documented public-interface so :shrug:. [Here's](https://stackoverflow.com/a/41672053) a stackoverflow thread on this difference. Here is the error in Python3 prior to the change:

```
self = <pg2kinesis.formatter.CSVPayloadFormatter object at 0x7f7d19cb1588>, change = Change(xid=1, table='public.blue', operation='Update', pkey='123456')

    def produce_formatted_message(self, change):
        fmt_msg = '{},{},{}'.format(CSVFormatter.VERSION, CSVFormatter.TYPE,
>                                   json.dumps(change.__dict__))
E       AttributeError: 'Change' object has no attribute '__dict__'

pg2kinesis/formatter.py:161: AttributeError
===================================================
```